docs/content/reference/routing-configuration/http/router/rules-and-priority.md

```diff
@@ -0,0 +1,288 @@
+
+It only matches the request client IP and does not use the `X-Forwarded-For` header for matching.
+
+| Behavior                                                        | Rule                                                                    |
+|-----------------------------------------------------------------|:------------------------------------------------------------------------|
+| Match requests coming from a given IP (IPv4). | ```ClientIP(`10.76.105.11`)``` |
+| Match requests coming from a given IP (IPv6). | ```ClientIP(`::1`)``` |
+| Match requests coming from a given subnet (IPv4). | ```ClientIP(`192.168.1.0/24`)``` |
+| Match requests coming from a given subnet (IPv6). | ```ClientIP(`fe80::/10`)``` |
+
+### RuleSyntax
+
+In Traefik v3 a new rule syntax has been introduced ([migration guide](../../../../migration/v3.md)). the `ruleSyntax` option allows to configure the rule syntax to be used for parsing the rule on a per-router basis. This allows to have heterogeneous router configurations and ease migration.
+
+The default value of the `ruleSyntax` option is inherited from the `defaultRuleSyntax` option in the install configuration (formerly known as static configuration). By default, the `defaultRuleSyntax` static option is v3, meaning that the default rule syntax is also v3
+
+#### Configuration Example
+
+The configuration below uses the [File Provider (Structured)](../../../install-configuration/providers/others/file.md) to configure the `ruleSyntax` to allow `Router-v2` to use v2 syntax, while for `Router-v3` it is configured to use v3 syntax.
+
+```yaml tab="Structured (YAML)"
+## Dynamic configuration
+http:
+  routers:
+    Router-v3:
+      rule: HostRegexp(`[a-z]+\\.traefik\\.com`)
+      ruleSyntax: v3
+    Router-v2:
+      rule: HostRegexp(`{subdomain:[a-z]+}.traefik.com`)
+      ruleSyntax: v2
+```
+
+```toml tab="Structured (TOML)"
+## Dynamic configuration
+[http.routers]
+  [http.routers.Router-v3]
+    rule = "HostRegexp(`[a-z]+\\.traefik\\.com`)"
+    ruleSyntax = v3
+  [http.routers.Router-v2]
+    rule = "HostRegexp(`{subdomain:[a-z]+}.traefik.com`)"
+    ruleSyntax = v2
+```
+
+```yaml tab="Labels"
+services:
+  my-container:
+    #...
+    labels:
+      - "traefik.http.routers.Router-v3.rule=HostRegexp(`[a-z]+\\.traefik\\.com`)"
+      - "traefik.http.routers.Router-v3.ruleSyntax=v3"
+      - "traefik.http.routers.Router-v2.rule=HostRegexp(`{subdomain:[a-z]+}.traefik.com`)"
+      - "traefik.http.routers.Router-v2.ruleSyntax=v2"
+```
+
+```json tab="Tags"
+  {
+    "Name": "Router-v3",
+    "Tags": [
+      "traefik.http.routers.Router-v3.rule=HostRegexp(`[a-z]+\\.traefik\\.com`)",
+      "traefik.http.routers.Router-v3.ruleSyntax=v3"
+    ]
+  },
+  {
+    "Name": "Router-v2",
+    "Tags": [
+      "traefik.http.routers.Router-v2.rule=HostRegexp(`{subdomain:[a-z]+}.traefik.com`)",
+      "traefik.http.routers.Router-v2.ruleSyntax=v2"
+    ]
+    // ...
+  }
```

Comment:
```suggestion
  {
    "Name": "my-service",
    "Tags": [
      "traefik.http.routers.Router-v3.rule=HostRegexp(`[a-z]+\\.traefik\\.com`)",
      "traefik.http.routers.Router-v3.ruleSyntax=v3"
      "traefik.http.routers.Router-v2.rule=HostRegexp(`{subdomain:[a-z]+}.traefik.com`)",
      "traefik.http.routers.Router-v2.ruleSyntax=v2"
    ]
    // ...
  }
```

I would simplify all Tags configuration examples that are similar to this one in this way.